scripts/mkstatic.py

```diff
@@ -667,11 +667,16 @@ def parse(self, make_call_tree=False):
         suite_xml = tree.getroot()
         self._name = suite_xml.get('name')
         # Validate name of suite in XML tag against filename; could be moved to common.py
-        if not (os.path.basename(self._sdf_name) == 'suite_{}.xml'.format(self._name)):
-            logging.critical("Invalid suite name {0} in suite definition file {1}.".format(
-                                                               self._name, self._sdf_name))
-            success = False
-            return success
+        if not (os.path.basename(self._sdf_name) == '{}.xml'.format(self._name)):
+            if (os.path.basename(self._sdf_name) == 'suite_{}.xml'.format(self._name)):
+                logging.debug("Parsing suite using legacy naming convention")
```

Comment:
Same here, should this be warning? Or leave it as debug if it throws a warning in the first place anyway?

Reply:
That was my thinking: I wanted a message here, but since it's already mentioned in a higher-priority message earlier I figured debug-level was appropriate here.